### ISSUE TITLE
Is the ability to publish the changelog to a PR a thing? 

### ISSUE THREAD

**Issue by Kimba666** [intuit/auto#1558]
**Is your feature request related to a problem? Please describe.**
In my team's dev environments, there is branch protection that does not allow anything to get merged to dev or master without being an approved PR. I'd make an exception for this tool but whenever there is a merge to dev or master, the instance rebuilds so it would trigger a 2nd rebuild every single deploy whenever the changelog published itself.

**Describe the solution you'd like**
Is there a command or some way that I can have the changelog file added to a Pull Request? I wrote a github action that automates what I want whenever there is a push to the dev branch. I'm wondering if there's a way to apply the changelog file as a commit that gets added to the pull request that triggered the job to run, instead of directly to the branch itself.

-Job is triggered to run when Pull Request is made on the dev branch
-job runs to generate changelog and bumps app version
-job publishes the changelog.md file to the pull request that triggered the job to start.
**Describe alternatives you've considered**

**Additional context**

**Comment by hipstersmoothie** [comment]
I think a better strategy would be to use `auto changelog --dry-run` to generate the changelog and `auto pr-body` to update the PR body with the changelog. this way we aren't adding a bunch of extra commits. Thoughts?

**Comment by Kimba666** [comment]
I think that's actually an excellent idea, I'm gonna do some testing locally and if I get the desired results I'll close this issue, if not I'll let you know what the outcome was.

**Comment by Kimba666** [comment]
I tinkered with this method in a few different ways and didn't get results I was satisfied with. Since then I've decided to use a different method. 

Is there a way to have the Changelog.md file that is generated by the 'yarn auto latest' command committed to a different repository from the one that it generates the changelog for? 

Example: Lets say we're in Repo A, and a trigger occurs that generates a changelog.md file for Repo A, but commits it to Repo B. How can this be done?
